### PR TITLE
Display Notification for the user

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5464

## Description

To be removed in https://wikia-inc.atlassian.net/browse/CAKE-5470

## Reviewers

@Wikia/cake

### CODE REVIEW

File: app/services/update-policy-links.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    + ' create a single privacy policy, terms of use and terms of sale'
+    + ' across all of our properties. These changes went into effect on January 1, 2020.',
+  updatePolicyKey: 'fandom-update-policy-links',
+  storageTrueValue: '1',
+  // Keep it for a year, it's more than enough
+  localStorageTTL: 60 * 60 * 24 * 365,
+
+  shouldShowPolicyNotification() {
+    return localStorageConnector.getItem(
+      this.updatePolicyKey,
```
Comment (mklucsarits) [inline app/services/update-policy-links.js:24]: This doesn't cause a syntax error? (comma before closing parenthesis)

Comment (vforge) [reply app/services/update-policy-links.js:24]: Nope

**Review by mklucsarits — APPROVED** [review]

File: app/services/update-policy-links.js (hunk 1 above)
Comment (cschucode) [reply app/services/update-policy-links.js:24]: looks weird.

**Comment by cschucode** [comment]
Looks like there's a hangup with the build.

**Pull request merged**